### PR TITLE
Refined strategy loader

### PR DESCRIPTION
**Pull request by mikepepping**
- Removed redundant file io
- added new meaningful errors
- changed create_strategy to export_strategy

### CODE REVIEW

File: core/strategy_loader.py [hunk 1]
```diff
@@ -18,13 +18,13 @@ def find_and_load(self):
...
 
     def _import_from_file(self, file_name):
-        with open(join(self.strategies_dir, file_name)) as file:
-            content = file.read()
-        
         package = re.sub("\.\/", "", self.strategies_dir)
         module_name = re.sub("\.py", "", file_name)
         new_module = import_module(package+"."+module_name,".")
-        self.strategies.append(new_module.create_strategy())
+        if "export_stragegy" in dir(new_module):
```
Comment (Catsuko) [inline core/strategy_loader.py]: stragegy ! ? !

File: strategies/test.py [hunk 2]
```diff
@@ -12,5 +12,5 @@ def place_token(self, token, board):
                 return x
         return -1
 
-def create_strategy():
+def export_stragegy():
```
Comment (Catsuko) [inline strategies/test.py]: typo as above

**Review by Catsuko — CHANGES_REQUESTED** [review]
Fix typos

File: core/strategy_loader.py (hunk 1 above)
Comment (mikepepping) [reply core/strategy_loader.py]: fixed!

File: strategies/test.py (hunk 2 above)
Comment (mikepepping) [reply strategies/test.py]: fixed!

**Review by mikepepping — COMMENTED** [review]
Typos fixed

**Review by Catsuko — APPROVED** [review]

**Pull request merged**